### PR TITLE
V2 restore libpod.Shutdown() when exiting podman commands

### PR DESCRIPTION
**Pull request by jwhonce**
Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: pkg/domain/infra/abi/images.go [hunk 1]
```diff
@@ -549,3 +549,8 @@ func (ir *ImageEngine) Remove(ctx context.Context, images []string, opts entitie
 
 	return
 }
+
+// Shutdown Libpod engine
```
Comment (mheon) [inline pkg/domain/infra/abi/images.go:553]: One of these (containers/images shutdown) is guaranteed to error, duplicate shutdown will never work.

**Comment by TomSweeneyRedHat** [comment]
LGTM.  I'd like to get a head nod from @mheon to make sure his concern was addressed.

**Pull request merged**